mosdef_cassandra/runners/runners.py

```diff
@@ -92,66 +95,80 @@ def run(system, moveset, run_type, run_length, temperature, **kwargs):
     _run_cassandra(cassandra, inp_file, log_file)
 
 
-def restart(system, moveset, run_type, run_length, temperature, **kwargs):
-    """Restart a Monte Carlo simulation with Cassandra
+def restart(restart_from=None, run_name=None, run_length=None, run_type=None):
+    """Restart a Monte Carlo simulation from a checkpoint file with Cassandra
+
+    The function requires the following in the working directory. These items
+    would have all been generated for the original run:
+        * Cassandra input (.inp) file named {restart_from}.inp
+        * Cassandra checkpoint file (.chk) name {restart_from}.out.chk
+        * MCF files for each species
+        * Fragment libraries for each species
+
+    The maximum translation, rotation, and volume move sizes are read from
+    the checkpoint file. Similarly, the starting structure is taken from the
+    checkpoint file. If the "restart_name" is not provided or if the
+    "run_name" is the same as "restart_name", ".rst.N" will be appended to the
+    "run_name".
 
-    This function is used to restart a Cassandra simulation from a
-    checkpoint file. No new MCF files are written to disk; the function
-    assumes they already exist. No new fragment libraries are generated,
-    again, these should already exist from the original run. The maximum
-    translation, rotation, and volume move sizes are read from the checkpoint
-    file so the values specified in mc.Moves are **not** used. Similarly,
-    the starting structure is taken from the checkpoint file. The
-    keyword argument "restart_name" should specify the old "run_name",
-    i.e., the "run_name" that you are restarting from. If the "restart_name"
-    is not provided or if the "run_name" is the same as "restart_name",
-    "-rst" will be appended to the "run_name".
+    If you wish to extend a simulation you will need to specify the _total_
+    number of simulation steps desired with the run_length option. For example,
+    if your original run was 1e6 MC steps, but you wish to extend it by an
+    additional 1e6 steps, use run_length=2000000.
 
     Parameters
     ----------
-    system : mosdef_cassandra.System
-        the System to simulate
-    moveset : mosdef_cassandra.MoveSet
-        the MoveSet to simulate
-    run_type : "equilibration" or "production"
+    restart_from: str, optional, default=None
+        name of run to restart from; if None, searches current
+        directory for Cassandra inp files
+    run_length: int, optional, default=None
+        total length of the MC simulation; if None, use original simulation length
+    run_name: str, optional, default=None
+        name of this run; if None, appends "_rst" to run_name
```

Comment:
Line 111 says that `.rst.` will be appended to a filename but 127 says `_rst` will be appended.  `_rst` is the string that is appended to a restart file name in #90.  I don't have a preference either way, but just want to be consistent.

Reply:
Good catch. I have moved to `.rst.` so I can use a sequential numbering scheme, `.rst.001.`, `.rst.002.`, ...